packages/server/src/fhir/operations/execute.test.ts

```diff
@@ -768,3 +844,19 @@ describe('Execute', () => {
     });
   });
 });
+
+function parseJwt(token: string): Record<string, string> {
```

Comment:
Surprised we don't have a util for this in core. Might be a good followup?

Reply:
We do have a util for this in core:  `parseJWTPayload` in `packages/core/src/jwt.ts`

Reply:
I apologize for inconsistent capitalization... That function is from early 2021 before we got OCD about such things.

Reply:
Ah yeah I thought that was a thing